db-demo-product/product.json

```diff
@@ -27,8 +27,8 @@
       "data": {
```

Comment:
I think you can remove the 'maven-dependency' block ... only the import is required. otherwise, your currently selected project will get a dependency to the demo.